users/app.py

```diff
@@ -60,5 +54,33 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def find_authorization_in_db(username, users_collection):
+    """Queries the db to find authorization of the given user.
+
+    Documents in the users collection should look like
+        {"username": "cmei4444",
+        "name": "Carolyn Mei",
+        "is_organizer": True}
+    """
+    if users_collection.count_documents({"username": username}) == 0:
+        return False  # user not found
+    first_user = users_collection.find_one({"username": username})
```

Comment:
It looks like `find_one` will return `None` if there are no matching documents, so you shouldn't need to call `count_documents` above. (This is more efficient on the database side, as it will do only one search.)

Reply:
Done https://github.com/knative-portability/large-events/pull/2/commits/249cd4512fc9efe0a420fbecd5200be21b73e141